### PR TITLE
EFA-572 Approach to fix "defective libraries" warning on Play Store

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java [hunk 1]
```diff
@@ -288,8 +288,8 @@ public void onClick(View view) {
     }
 
     public void showCryptoDialog(CryptoMode currentCryptoMode) {
-        CryptoSettingsDialog dialog = CryptoSettingsDialog.newInstance(currentCryptoMode);
-        dialog.show(activity.getFragmentManager(), "crypto_settings");
+        //CryptoSettingsDialog dialog = CryptoSettingsDialog.newInstance(currentCryptoMode);
```
Comment (Stereodark) [inline k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java:291]: will this then be an empty dialog?

**Review by Stereodark — COMMENTED** [review]
looks good to me, a big refactoring to comply to the store policies, I think it looks good from my perspective.

File: k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java (hunk 1 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java:291]: It is actually obsolete code from K-9 that is not used. So I just commented code that was hindering the refactor itself.

Comment (Stereodark) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java:291]: I guess then we can remove the entire function as it does nothing

**Review by Stereodark — APPROVED** [review]
looks fine to me

File: k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java (hunk 1 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java:291]: alright, removed, thanks.

**Pull request merged**